From: Antonin Kral <a.kral@bobek.cz>
Date: Thu, 16 Feb 2012 09:48:29 +0100
Subject: =?UTF-8?q?remove-pl-suffix=0Aremove=20.pl=20suffix=20from=20sqlgrey?=
 =?UTF-8?q?-logstats.pl?=

---
 Makefile |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 5bceb02..5506982 100644
--- a/Makefile
+++ b/Makefile
@@ -20,8 +20,8 @@ update-version:
 	cat sqlgrey.spec | sed 's/^%define ver  .*/%define ver  $(VERSION)/' > sqlgrey.spec.new
 	mv sqlgrey.spec.new sqlgrey.spec
 	cat sqlgrey-logstats.pl | sed 's/^my $$VERSION = .*;/my $$VERSION = "$(VERSION)";/' > sqlgrey-logstats.pl.new
-	mv sqlgrey-logstats.pl.new sqlgrey-logstats.pl
-	chmod a+x sqlgrey-logstats.pl
+	mv sqlgrey-logstats.pl.new sqlgrey-logstats
+	chmod a+x sqlgrey-logstats
 
 use_dbcluster:
 	cat sqlgrey | sed 's/^use DBI;/use DBIx::DBCluster;/' > sqlgrey.new
@@ -50,7 +50,7 @@ install: all
 	$(INSTALL) -d -m 755 $(BINDIR)
 	$(INSTALL) -m 755 sqlgrey $(SBINDIR)
 	$(INSTALL) -m 755 update_sqlgrey_config $(SBINDIR)
-	$(INSTALL) -m 755 sqlgrey-logstats.pl $(BINDIR)
+	$(INSTALL) -m 755 sqlgrey-logstats $(BINDIR)
 	$(INSTALL) -m 644 etc/sqlgrey.conf $(CONFDIR)
 	$(INSTALL) -m 644 etc/clients_ip_whitelist $(CONFDIR)
 	$(INSTALL) -m 644 etc/clients_fqdn_whitelist $(CONFDIR)
