From: Antonin Kral <a.kral@bobek.cz>
Date: Thu, 16 Feb 2012 09:56:43 +0100
Subject: =?UTF-8?q?add-logstats-manpage=0Aadd=20generation=20of=20logstats=20?=
 =?UTF-8?q?manpage?=

---
 Makefile |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index 043504c..a4b7eff 100644
--- a/Makefile
+++ b/Makefile
@@ -36,9 +36,10 @@ use_dbi:
 
 manpage:
 	perldoc -u sqlgrey | pod2man -n sqlgrey > sqlgrey.1
+	perldoc -u sqlgrey-logstats | pod2man -n sqlgrey-logstats | sed 's/sqlgrey.*-logstats\.pl/sqlgrey-logstats/g' > sqlgrey-logstats.1
 
 clean:
-	rm -f sqlgrey.1
+	rm -f sqlgrey.1 sqlgrey-logstats.1
 	rm -f *~ .#* init/*~ etc/*~
 
 install: all
@@ -59,6 +60,7 @@ install: all
 	$(INSTALL) -m 644 etc/smtp_server.regexp $(CONFDIR)
 	$(INSTALL) -m 644 etc/README $(CONFDIR)
 	$(INSTALL) -m 644 sqlgrey.1 $(MANDIR)
+	$(INSTALL) -m 644 sqlgrey-logstats.1 $(MANDIR)
 
 rh-install: install
 	$(INSTALL) init/sqlgrey $(INITDIR)
